evergreen/install_baas.sh

```diff
@@ -227,7 +227,7 @@ YARN="$WORK_PATH/yarn/bin/yarn"
 if [[ ! -x "$YARN" ]]; then
     echo "Getting yarn"
     mkdir yarn && cd yarn
-    $CURL -LsS https://s3.amazonaws.com/stitch-artifacts/yarn/latest.tar.gz | tar -xz --strip-components=1
+    $CURL -LsS https://yarnpkg.com/latest.tar.gz | tar -xz --strip-components=1
```

Comment:
do we want to put this in our S3 bucket that we use for other artifacts?

Reply:
I was wondering if we should pin the version on S3. The server team is updating their tests to use the official download path.